components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/FirefoxAccount.kt

```diff
@@ -144,7 +144,7 @@ class FirefoxAccount internal constructor(
         accessType: AccessType
     ) = withContext(scope.coroutineContext) {
         handleFxaExceptions(logger, "authorizeOAuthCode", { null }) {
-            val params = AuthorizationParams(clientId, scopes, state, accessType.msg)
+            val params = AuthorizationParameters(clientId, scopes.toList(), state, accessType.msg, null, null, null)
```

Comment:
slightly offtopic for this PR, and we already discussed this in slack: it took me a bit of clicking around to figure out that the missing params here are for the PKCE code exchange. I think it's only documented at https://github.com/mozilla/application-services/blob/d77600e32c19cbeebaf1045a1b9b49476530e770/components/fxa-client/src/internal/oauth.rs#L194

Also, filed https://github.com/mozilla-mobile/android-components/issues/9863 to remove this from the API surface for now.

Reply:
I guess we could use named argument syntax here to make things bit clearer, e.g. `AuthorizationParameters(..., pkceParams=null, keysJwk=null)`.

I also expect a future release to support default values for these arguments which might help clean it up a bit more.